app/qml/map/MapWrapper.qml

```diff
@@ -450,7 +450,7 @@ Item {
     .arg( __inputUtils.formatNumber( _positionKit.accuracy ) )
     link: __inputHelp.gpsAccuracyHelpLink
 
-    showWarning: shouldShowAccuracyWarning
+    showWarning: true//shouldShowAccuracyWarning
```

Comment:
I guess you did not want to commit this change?

Reply:
Yes - sorry about it